index.html

```diff
@@ -38,6 +38,8 @@ <h1 id="title"> {{ appCtrl.Event.name }} </h1>
         </md-content>
     </div>
 
+    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```

Comment:
why does this needs to be here? I don't see the point. It be injected via another component.